contracts/Tokens/VTokens/VTokenInterfaces.sol

```diff
@@ -3,7 +3,16 @@ pragma solidity ^0.5.16;
 import "../../Comptroller/ComptrollerInterface.sol";
 import "../../InterestRateModels/InterestRateModel.sol";
 
-contract VTokenStorage {
+interface IProtocolShareReserve {
```

Comment:
Could we use the new npm package `@venusprotocol/protocol-reserve`?

Reply:
`@venusprotocol/protocol-reserve` contains 0.8.13 sol version of IProtocolShareReserve which is incompatible with this contracts.